specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -532,6 +532,102 @@
           }
         ]
       }
+    },
+    "/providers/Microsoft.Subscription/policies/default": {
```

Comment:
Should we call it "tenantPolicies"? That way we can reserve the name "policies" for policies scoped to a subscription, should we ever have a scenario for that.

Reply:
Makes sense to me.

---
In reply to: [554133578](https://github.com/Azure/azure-rest-api-specs/pull/11672#discussion_r554133578) [](ancestors = 554133578)

Reply:
we have billingPolicies  which would be 
/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/providers/Microsoft.Subscription/Policies/default